### DISCUSSION TITLE
Is there any way to disable authentication for admin user?

### DISCUSSION THREAD

**Discussion by ep4sh** [argoproj/argo-cd#11212] [Q&A]
Dear community, 

I walked over the helm chart values, but didn't find the way how to disable authentication for admin user from out-of-the-box

Please suggest if it's possible.

Thanks a lot,
P.

**Comment by alexef** [comment]
https://github.com/argoproj/argo-cd/blob/master/docs/faq.md#how-to-disable-admin-user is this what you are looking for? 

The helm chart [allows you](https://github.com/argoproj/argo-helm/tree/main/charts/argo-cd#argo-cd-configs) to set the argocd-cm content.

**Reply by ep4sh** [reply]
Hi @alexef , thank you for your reply, but it seems that is not my case.
I just want to auth as admin without the password (not to disable the admin user)

**Reply by yonatanaftali** [reply]
@ep4sh have you figured it out? I'm trying to do the same thing.

**Reply by ep4sh** [reply]
We've just done auth with SSO (keycloak)

**Comment by crenshaw-dev** [comment]
You can enable "anonymous mode" and then make the default role admin. I obviously wouldn't recommend that in a production system.
